rlp/raw_test.go

```diff
@@ -60,15 +60,43 @@ func TestCountValues(t *testing.T) {
 	}
 }
 
-func TestSplitTypes(t *testing.T) {
-	if _, _, err := SplitString(unhex("C100")); err != ErrExpectedString {
-		t.Errorf("SplitString returned %q, want %q", err, ErrExpectedString)
+func TestSplitString(t *testing.T) {
+	tests := []struct {
+		input string
+		err   error
+	}{
```

Comment:
You can simplify it now that you split them into two
```suggestion
	tests := []sting{
```